### PR TITLE
Ncl 1461

### PR DESCRIPTION
**Pull request by vibe13**
Reduce code duplication in creation of BCs

### CODE REVIEW

**Comment by vibe13** [comment]
@alexcreasy @pgier please review. If we get this in, then we have only one place to debug for the js error you were encountering

File: ui/app/configuration/directives/pnc-configuration-create-panel/pncConfigurationCreatePanel.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  var module = angular.module('pnc.configuration');
+
+  module.directive('pncConfigurationCreatePanel', [
+    function () {
+      
+      function PncConfigurationCreateController($log, $state, $filter, $scope, Notifications, 
+    		  EnvironmentDAO, ProjectDAO, ProductDAO, BuildConfigurationDAO, BuildConfigurationSetDAO) {
+
+    	// Selection of Product Versions.
+        $scope.productVersions = {
```
Comment (alexcreasy) [inline ui/app/configuration/directives/pnc-configuration-create-panel/pncConfigurationCreatePanel.js]: The indentation needs fixing here

File: ui/app/configuration/directives/pnc-configuration-create-panel/pncConfigurationCreatePanel.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    	  selected: []
```
Comment (alexcreasy) [inline ui/app/configuration/directives/pnc-configuration-create-panel/pncConfigurationCreatePanel.js]: The indentation goes a bit wrong here, this line needs to be indented twice more and the indentation is broken from this line onwards.

**Comment by alexcreasy** [comment]
@vibe13 Great! If you fix the indentation in pncConfigurationCreatePanel.js this gets a +1 from me

**Comment by vibe13** [comment]
@alexcreasy  I have now applied the 2-spaces indentation as in the other js pages

**Comment by alexcreasy** [comment]
@vibe13 np, it's good to get this one fixed finally!

**Pull request merged**